pkg/pillar/types/downloadertypes.go

```diff
@@ -36,7 +36,7 @@ func (config DownloaderConfig) LogCreate(logBase *base.LogObject) {
 		return
 	}
 	logObject.CloneAndAddField("target", config.Target).
-		AddField("datastore-id", config.DatastoreID).
+		AddField("datastore-id", config.DatastoreIDsList[0]).
```

Comment:
For the logs I think it would make sense to introduce a function which produces a concatentated string of all of the elements of IDsList.

Reply:
Do we limit this list? If we a bunch of datastores then we have a looooong string. Already 3 uuids would look ugly.

Reply:
Not for human consumption so shouldn't matter. If we have lots we might exceed the (2kbyte?) maxpathlen for a filename, but I'm not worried about that based on the intended usage.

Alternative would be have the key (used for logs and for the ResolveConfig) be a sha256 of all of the datastore IDs, but that makes it impossible to find a particular UUID in the logs.

Reply:
I thought that this logobject is what we see as Json in kibana logs. So I might be wrong. Anyway, already changed that. Need to push latest changes